src/main/java/com/dataguise/saas/controllers/MetadataController.java

```diff
@@ -0,0 +1,98 @@
+package com.dataguise.saas.controllers;
+
+import com.dataguise.saas.dto.ColumnViewUIFormatterDTO;
+import com.dataguise.saas.dto.SensitiveExpressionDTO;
+import com.dataguise.saas.dto.SourceTreeViewUIFormatterDTO;
+import com.dataguise.saas.exception.BadGatewayException;
+import com.dataguise.saas.exception.BadRequestException;
+import com.dataguise.saas.exception.InternalServerException;
+import com.dataguise.saas.service.ProtectionTaskService;
+import com.dataguise.saas.service.SourceHierarchyExplorerService;
+import com.dataguise.saas.service.TaskBuilderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+
+@CrossOrigin
+@RestController
+@ConfigurationProperties(prefix = "dgsecure")
+@PropertySource("classpath:application.properties")
+@RequestMapping(value = "/api/dbms")
+
+public class MetadataController {
+
+    @Autowired
+    ProtectionTaskService protectionTaskService;
+
+    @Autowired
+    SourceHierarchyExplorerService sourceHierarchyExplorerService;
+
+    @Autowired TaskBuilderService taskBuilderService;
+
+
+    /**
+     * List detection and protection sources on scheduler page while creating a schedule in formatted structure.
+     *
+     * @param connectionId
+     * @return
+     * @throws BadGatewayException
+     */
+    @RequestMapping(value = {"/connections", "/connections/{connectionId}"}, method = RequestMethod.GET)
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocMethodCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/controllers/MetadataController.java L45
    
    Expected @return tag.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-228762442>*